### PR TITLE
Updated Rent vs Buy Calculator to take input and output text.

### PR DESCRIPTION
**Pull request by hernandezv0110**

### CODE REVIEW

**Review by aditirai-12 — APPROVED** [review]

File: portfolio/src/main/webapp/buy_rent_logic.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    {
+        total_rent = total_rent + year_rent;
+        year = year + 1;
+    }
+    
+    const result_text = document.getElementById('calcResult');
+    const advice_text = document.getElementById('calcAdvice');
+    result_text.innerText = 'After ' + year + ' years, you would have paid the same amount for rent and a home!';
+    
+    if (num_years >= year)
```
Comment (jdtyner) [inline portfolio/src/main/webapp/buy_rent_logic.js:21]: I think you should use more descriptive variable names here. It looks like `year` is the number of years before breakeven, and `num_years` is the number of years they expect to live in the location?

File: portfolio/src/main/webapp/buy_rent_logic.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    {
+        advice_text.innerText = 'Since you expect to stay at the property for ' + num_years + ', then buying a home might be a good option.';
```
Comment (jdtyner) [inline portfolio/src/main/webapp/buy_rent_logic.js:23]: I think you might want to be more authoritative and say renting/buying "might be a _better_ option.". Saying they "might be a good option" is something that could be true for both.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -33,7 +33,7 @@ <h1>Welcome to BetterBudget!</h1>
                 <a class="nav-link" href="emergencyFund.html">Emergency Fund</a> <!--To Emergency Fund Calculator-->
               </li>
               <li class="nav-item">
-                <a class="nav-link" href="#"> Rent vs. Buy</a>
+                <a class="nav-link" href="buy_rent.html"> Rent vs. Buy</a> <!--To Rent vs Buy Calculator-->
```
Comment (jdtyner) [inline portfolio/src/main/webapp/index.html:36]: Can you update this on [the other navbar](https://github.com/jdtyner/sps22-team6/blob/d6aebf1dbd0362c8f64894c2da6a7dd6223b037a/portfolio/src/main/webapp/income-expense.html#L33) too

**Review by jdtyner — APPROVED** [review]

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (LivsWorld) [reply portfolio/src/main/webapp/index.html:36]: I can update that

**Review by LivsWorld — APPROVED** [review]

**Pull request merged**